src/test/java/org/mule/extension/ftp/FtpWriteTestCase.java

```diff
@@ -199,4 +221,27 @@ private String doWriteOnExistingFile(FileWriteMode mode) throws Exception {
     doWrite(filePath, HELLO_WORLD, mode, false);
     return toString(readPath(filePath).getPayload().getValue());
   }
+
+  public static InputStream getContentStream() {
+    return (new InputStream() {
+
+      String text = "Hello World!";
+      char[] textArray = text.toCharArray();
+      int index = -1;
+
+      @Override
+      public int read() throws IOException {
+        try {
+          Thread.sleep(10);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2925))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-ftp-connector&pullRequest=101&issues=AWkMI-sJotIXnmsUtDdh&open=AWkMI-sJotIXnmsUtDdh)

[ISSUE_KEY]: AWkMI-sJotIXnmsUtDdh